page.js

```diff
@@ -15,13 +17,17 @@ class Page {
         });
 
         if (this.sleeping) {
-            this.screen.driver.init();
+            if (this.color_depth == common.BW) {
+                this.screen.driver.init();
```

Comment:
I think it might be a bit cleaner to try a polymorphic solution, instead of using the `color_depth` flag.  It would work the same way as the `renderPNG` method exposed by each device. See my other comment for more details

Reply:
Done, please check 69de6b0